forgerock-openbanking-config/forgerock-openbanking-git-config/application-native.yml

```diff
@@ -61,6 +61,10 @@ rs-discovery:
   base-url: https://${rs-discovery.hostname}:8074
   read-write-api:
     version: 3.1.8
+  apis:
```

Comment:
Now you have the api definition in here, and you should remove it. You can also remove the flag to turn it on because the flag should be set to true and the definition of the endpoint API specified in application.yaml

Reply:
done!